controllers/operands/kubevirtConsolePlugin.go

```diff
@@ -55,6 +52,10 @@ func newKvUiPluginCRHandler(_ log.Logger, Client client.Client, Scheme *runtime.
 }
 
 func NewKvUiPluginDeplymnt(hc *hcov1beta1.HyperConverged) (*appsv1.Deployment, error) {
+	kvUiPluginImage, varExists := os.LookupEnv(hcoutil.KvUiPluginImageEnvV)
+	if !varExists {
+		return nil, errors.New(fmt.Sprintf("%s env var not found", hcoutil.KvUiPluginImageEnvV))
```

Comment:
This is a problem because in this case, HCO is in error loop that can't be resolved.

I think there are three options here:
1. Check it on boot and exit with error. That way a wrong image will fail in any basic test
2. only log here, but don't return error.
3. some combination of the first two

Reply:
implemented both approaches.